lib/messageUtils.ts

```diff
@@ -133,3 +123,19 @@ export function getDashboardIdForTemplate(template: string) {
     return "1806";
   }
 }
+
+/**
+ * @returns the submission timestamp date filter to be used in the Looker dashboard links
+ */
+export function getSubmissionTimestampDateFilter(startDate?: string | null, endDate?: string | null): string {
```

Comment:
This seems like it also wants to live in `looker.ts` for separation-of-concern reasons.